ts/components/AssistanceFormWebView.tsx

```diff
@@ -0,0 +1,75 @@
+import { StyleSheet, View } from "react-native";
+import React, { createRef, FC, useState } from "react";
+import WebView from "react-native-webview";
+import { AVOID_ZOOM_JS, closeInjectedScript } from "../utils/webview";
+import { RefreshIndicator } from "./ui/RefreshIndicator";
+import GenericErrorComponent from "./screens/GenericErrorComponent";
+import { withLightModalContext } from "./helpers/withLightModalContext";
+
+const URI = "http://localhost:5000/";
+
+const styles = StyleSheet.create({
+  refreshIndicatorContainer: {
+    position: "absolute",
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0,
+    backgroundColor: "rgba(255,255,255,0.5)",
+    justifyContent: "center",
+    alignItems: "center",
+    zIndex: 1000
+  },
+  genericError: {
+    flex: 1,
+    position: "absolute",
+    left: 0,
+    right: 0,
+    top: 0,
+    bottom: 0
+  }
+});
+
+/** This component is basically a webview that loads an url showing assistance form */
```

Comment:
How do you think about removing implementation details on the component description?
```suggestion
/** This component loads an url and show the assistance form */
```